modules/ROOT/pages/upgrade-from-community-to-a-subscription-edition.adoc

```diff
@@ -1,59 +1,112 @@
+. Remove Bonita Community Web App from `<bonita-installation>/server/webapps`: both `bonita.war` and the auto-extracted folder with the same name `bonita/`
+. Copy the Bonita Enterprise Web App (extracted from Bonita Enterprise bundle) where the Community version was: `<bonita-installation>/server/webapps/bonita.war`
+. ⚠ (Optionally rename the `<bonita-installation>` folder with a name that does not contain anymore the word `community`, for clarity)
+. Start the upgraded Bonita platform using `<bonita-installation>/start-bonita(.sh/.bat)` script
+
+
+=== Notes
 
-The Upgrade is now finished, you can verify that you are now running a subscription edition in the portal user interface, the dialog displayed from the top right "Settings / About" menu should indicate the correct Subscription edition.
+. If you remove bonita webapp folder while Tomcat is running, Tomcat server considers that the webapp should be completely
+unloaded and removed. It also deletes the webapp configuration file `<bonita-installation>/server/conf/Catalina/localhost/bonita.xml`, resulting in many errors in the log files. Hopefully, Bonita restores this file during next run of `start-bonita(.sh/.bat)` script
 
-Once you have started Bonita Subscription platform at least once, you can pull your Subscription configuration (`<install>/setup/setup(.sh/.bat) pull`)
-and change the Subscription-specific configuration files to match your needs. See xref:BonitaBPM_platform_setup.adoc[Bonita platform setup tool] for details about
+The upgrade is now finished, you can verify that you are now running an Enterprise edition in the portal user interface; the dialog displayed from the top right "Settings / About" menu should indicate the correct Enterprise edition.
+
+Once you have started Bonita Enterprise platform at least once, you can pull your Enterprise configuration (`<install>/setup/setup(.sh/.bat) pull`)
+and change the Enterprise-specific configuration files to match your needs. See xref:BonitaBPM_platform_setup.adoc[Bonita platform setup tool] for details about
 how to apply configuration changes.
+
+== Downgrade
+
+[WARNING]
+====
+Downgrading to Bonita Community edition means you will not be able to use Bonita Enterprise edition features anymore,
+so a prerequisite is to ensure you do not use any of these features before downgrading. Neither in your processes, nor in
+the platform settings.
+====
+
+=== Bonita studio
+
+To downgrade an Enterprise edition of Bonita Studio and its resources (such as process diagrams, data models), follow these steps:
+
+. xref:bonita-bpm-studio-installation.adoc[Install a Bonita  Studio] Community edition.
+. Export your resources from the Bonita Studio Enterprise edition to `.bos` files. We recommend that you keep these resource files as a backup.
+. Import your resource files into the Bonita Studio in Community edition.
+
+The downgrade is performed automatically and silently when importing resources into Bonita Studio Enterprise edition.
```

Comment:
Why precising "silently"?

Reply:
@dcoille It's the same as for the upgrade: the Studio does not issue and specific message in that case